counterpoll/main.py

```diff
@@ -382,6 +388,19 @@ def disable(ctx):
     fc_info['FLEX_COUNTER_STATUS'] = 'disable'
     ctx.obj.mod_entry("FLEX_COUNTER_TABLE", "FLOW_CNT_ROUTE", fc_info)
 
+
+@cli.command()
+@click.pass_context
+def delay(ctx):
+    """ Delays all counters for next boot """
+    config_db = ctx.obj
+
+    for key in config_db.get_keys('FLEX_COUNTER_TABLE'):
+        entry = config_db.get_entry('FLEX_COUNTER_TABLE', key)
+        entry['FLEX_COUNTER_DELAY_STATUS'] = 'true'
```

Comment:
What's the guarantee that `entry` here is a dict? Is there a chance of `entry['FLEX_COUNTER_DELAY_STATUS']` failing w/ entry being non dict? Or a failure such as `KeyError`?

Reply:
@vaibhavhd it's swsscommon API it returns a dict. No KeyError as this is a set operation - it creates a new key or overrides it's value